services/testing-framework/claudie_test.go

```diff
@@ -279,3 +329,21 @@ func checkIfManifestSaved(configID string, idType pb.IdType, c pb.ContextBoxServ
 		}
 	}
 }
+
+// cleanUp will delete manifest from claudie which will trigger infra deletion
+// it deletes a secret if claudie is deployed in k8s cluster
+// it calls for a deletion from database directly if claudie is deployed locally
+func cleanUp(id string, c pb.ContextBoxServiceClient) error {
+	if envs.Namespace != "" {
+		//delete secret from namespace
```

Comment:
```go
//abc
```
vs
```go
// abc
```
It is also advised that comments are written as full sentences ending in full stops.